project/BlendedActivemqBrokerstarter.scala

```diff
@@ -0,0 +1,19 @@
+import sbt._
+
+object BlendedActivemqBrokerstarter extends ProjectHelper {
```

Comment:
Wouldn't be `ProjectFactory` a better name for this class/trait? Also, the `helper` val below could be `projectSettings` or `config`.